lib/desired-caps.js

```diff
@@ -76,6 +76,9 @@ let commonCapConstraints = {
   chromeDriverPort: {
     isNumber: true
   },
+  chromedriverArgs: {
```

Comment:
it would be nice to have the cap documented elsewhere

Reply:
Ugh. **One** cap has snake case in "chromedriver"!

Reply:
yeah :-(